go/systests/teams_implicit_test.go

```diff
@@ -512,3 +514,40 @@ func TestResolveSBSConsolidatedTeamWithConflict(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, teamid2, lookupTeamID)
 }
+
+func TestCreateAndResolveEmailImpTeam(t *testing.T) {
```

Comment:
@heronhaye added another test to make sure mixed-case email SBS is covered

Assuming bob's mailserver is case-sensitive, this will make maintenanced not being able to send email to him because it will see "bob+someemail@noemail.keybase.io" instead of "BOB+someemail@noemail.keybase.io". But otherwise everything else works - adding that email, verifying it, making it public, and being let into an sbs team.

Reply:
Looks good